libraries/tools/kotlin-stdlib-gen/src/templates/Aggregates.kt

```diff
@@ -31,14 +31,22 @@ object Aggregates : TemplateGroupBase() {
         include(Maps, CharSequences, ArraysOfUnsigned)
     } builder {
         inline()
-        specialFor(ArraysOfUnsigned) { inlineOnly() }
+        specialFor(ArraysOfUnsigned) {
+            inlineOnly()
+            sample("samples.collections.Collections.Aggregates.all")
+        }
+        specialFor(Maps) {
+            sample("samples.collections.Collections.Aggregates.all")
+        }
+        specialFor(CharSequences) {
+            sample("samples.text.Strings.all")
+        }
 
         doc {
             """
             Returns `true` if all ${f.element.pluralize()} match the given [predicate].
             """
         }
-        sample("samples.collections.Collections.Aggregates.all")
```

Comment:
After changing the template files sources should be regenerated using `kotlin:libraries:tools:kotlin-stdlib-gen:run` Gradle task.

Reply:
Thank you for your review @qurbonzoda.
All requested changes has been done and template files sources have been regenerated.